### DISCUSSION TITLE
Adding More Join Types

### DISCUSSION THREAD

**Discussion by jasoula** [taozhi8833998/node-sql-parser#2032] [Ideas]
Hi,

as for now, there are only three JOIN types supported. Are there plans on supporting more join types?

**Comment by taozhi8833998** [comment]
@jasoula Please list the additional join types and their corresponding dialects, and I will try to add them.

**Reply by jasoula** [reply]
Thank you! For Postgres I am missing: Full Join, Cross Join, and the possibility to combine INNER and OUTER Keywords with Full, Right, Left
